### PR TITLE
Style and consistency fixes

### PR DESCRIPTION
**Pull request by psignoret**

### CODE REVIEW

File: centeredUi/README.md [hunk 1]
```diff
@@ -1,78 +1,77 @@
...
-    Note: It is recommended that you minify your CSS for a production environment  
+1. Download the ```ThemeCenterBrand.css``` file to your AD FS server, wherever you host your style sheets.
 
-2. Create a custom web theme using the following command in PowerShell: 
+    Note: It is recommended that you minify your CSS for a production environment.
 
-    ```New-AdfsWebTheme –Name custom -SourceName default –StyleSheet @{path=”c:\style\ThemeCenterBrand.css”}```
+2. Create a custom web theme using the following command in PowerShell:
+
+    ```New-AD FSWebTheme –Name custom -SourceName default –StyleSheet @{path="c:\style\ThemeCenterBrand.css"}```
```
Comment (bongiovimatthew-microsoft) [inline centeredUi/README.md]: PowerShell cmdlet needs to be New-AdfsWebTheme with no space in "Adfs"

File: centeredUi/README.md [hunk 2]
```diff
@@ -1,78 +1,77 @@
...
 
 3. Apply the new custom web theme using the following command in PowerShell:
 
-    ```Set-AdfsWebConfig -ActiveThemeName custom```
+    ```Set-AD FSWebConfig -ActiveThemeName custom```
```
Comment (bongiovimatthew-microsoft) [inline centeredUi/README.md]: PowerShell cmdlet needs to be Set-AdfsWebConfig with no space in "Adfs"

File: centeredUi/README.md [hunk 3]
```diff
@@ -1,78 +1,77 @@
...
-    
-    Note: It is *__highly__* recommended that you minify your ```paginatedOnload.js``` before including it in a production environment. There are many popular tools online 
-    for minifying JavaScript code. Two popular choices are [minifier.org](http://www.minifier.org/) and [JSCompress](https://jscompress.com/).
+    Note: It is *__highly__* recommended that you minify your ```paginatedOnload.js``` before including it in a production environment. There are many popular tools online for minifying JavaScript code. Two popular choices are [minifier.org](http://www.minifier.org/) and [JSCompress](https://jscompress.com/).
 
-2. Modify your existing custom web theme from the style sheet deployment to include the new JavaScript. In PowerShell: 
+2. Modify your existing custom web theme from the style sheet deployment to include the new JavaScript. In PowerShell:
 
-    ```Set-AdfsWebTheme –TargetName custom -AdditionalFileResource @{Uri=’/adfs/portal/script/onload.js’; path="c:\paginatedOnload.js"}```
+    ```Set-AD FSWebTheme –TargetName custom -AdditionalFileResource @{Uri="/AD FS/portal/script/onload.js"; path="c:\paginatedOnload.js"}```
```
Comment (bongiovimatthew-microsoft) [inline centeredUi/README.md]: PowerShell cmdlet needs to be Set-AdfsWebTheme with no space in "Adfs". 

Also, the URI needs to be "/adfs" not "/AD FS"

File: centeredUi/README.md [hunk 4]
```diff
@@ -1,78 +1,77 @@
...
 
-4. Apply the modified custom web theme using the following command in PowerShell: 
+3. Apply the modified custom web theme using the following command in PowerShell:
 
-    ```Set-AdfsWebConfig -ActiveThemeName custom```
+    ```Set-AD FSWebConfig -ActiveThemeName custom```
```
Comment (bongiovimatthew-microsoft) [inline centeredUi/README.md]: PowerShell cmdlet needs to be Set-AdfsWebTheme with no space in "Adfs".

**Comment by psignoret** [comment]
Sorry, my bad! Should be fixed now.

**Pull request merged**